recipes-extended/xen/files/libxl-iso-hotswap.patch

```diff
@@ -236,28 +230,28 @@ Index: xen-4.9.0/tools/libxl/libxl_internal.h
  _hidden char *libxl__device_libxl_path(libxl__gc *gc, libxl__device *device);
  _hidden int libxl__parse_backend_path(libxl__gc *gc, const char *path,
                                        libxl__device *dev);
-@@ -1765,6 +1766,9 @@ _hidden char *libxl__blktap_devpath(libx
-                                     libxl_disk_format format,
-                                     char *keydir);
+@@ -1764,6 +1765,9 @@ _hidden char *libxl__blktap_devpath(libx
+                                     const char *disk,
+                                     libxl_disk_format format);
```

Comment:
Why did you re-order the parameters and rename one? seems like unnecessary churn to me.

Reply:
I didn't, I'm not sure why git is showing it this way. May be because of the review fixes and patch updates. You can check all the patches here `https://github.com/mahantesh-ais/xenclient-oe/tree/prv4/`. The order and naming has always been 
`(libxl__gc *gc,`
 `const char *disk,`
 `libxl_disk_format format,`
 `char *keydir);`
and its the same now.